src/App.js

```diff
@@ -1,68 +1,51 @@
 import React from 'react';
 import UsersData from './data/users.json';
-
+import { User } from './components/User.js';
```

Comment:
We start variable with a capital letter for classes' constructors or Reacts components only